Upload stops after each file

``` python
[DEBUG] root 92: Version - 2.1.7
[DEBUG] pyupdater.client 58: PyUpdater Version 2.1.7
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.package_handler 94: Patch support enabled
[DEBUG] pyupdater.package_handler 94: Patch support enabled
[DEBUG] pyupdater.uploader 96: PLUGIN_NAMESPACE: pyupdater.plugins
[DEBUG] pyupdater.uploader 96: PLUGIN_NAMESPACE: pyupdater.plugins
[DEBUG] root 360: Requested uploader is not installed
[DEBUG] root 360: Requested uploader is not installed
[ERROR] root 362: Invalid upload plugin
[ERROR] root 362: Invalid upload plugin
[ERROR] root 363: Use "pyupdater plugins" to get a list of installed plugins
[ERROR] root 363: Use "pyupdater plugins" to get a list of installed plugins
[DEBUG] root 92: Version - 2.1.7
[DEBUG] pyupdater.client 58: PyUpdater Version 2.1.7
[DEBUG] root 92: Version - 2.1.7
[DEBUG] pyupdater.client 58: PyUpdater Version 2.1.7
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.package_handler 94: Patch support enabled
[DEBUG] pyupdater.package_handler 94: Patch support enabled
[DEBUG] pyupdater.uploader 96: PLUGIN_NAMESPACE: pyupdater.plugins
[DEBUG] pyupdater.uploader 96: PLUGIN_NAMESPACE: pyupdater.plugins
[DEBUG] root 360: Requested uploader is not installed
[DEBUG] root 360: Requested uploader is not installed
[ERROR] root 362: Invalid upload plugin
[ERROR] root 362: Invalid upload plugin
[ERROR] root 363: Use "pyupdater plugins" to get a list of installed plugins
[ERROR] root 363: Use "pyupdater plugins" to get a list of installed plugins
[DEBUG] root 92: Version - 2.1.7
[DEBUG] pyupdater.client 58: PyUpdater Version 2.1.7
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 52: Config Dir: /Users/ben/Projects/foo/.pyupdater
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.utils.storage 55: Config DB: /Users/ben/Projects/foo/.pyupdater/config.pyu
[DEBUG] pyupdater.package_handler 94: Patch support enabled
[DEBUG] pyupdater.package_handler 94: Patch support enabled
[DEBUG] stevedore.extension 157: found extension EntryPoint.parse('scp = scp_uploader:SCPUploader')
[DEBUG] stevedore.extension 157: found extension EntryPoint.parse('scp = scp_uploader:SCPUploader')
[DEBUG] paramiko.transport 1563: starting thread (client mode): 0xc04b950L
[DEBUG] paramiko.transport 1563: starting thread (client mode): 0xc04b950L
[DEBUG] paramiko.transport 1563: Local version/idstring: SSH-2.0-paramiko_2.0.2
[DEBUG] paramiko.transport 1563: Local version/idstring: SSH-2.0-paramiko_2.0.2
[DEBUG] paramiko.transport 1563: Remote version/idstring: SSH-2.0-OpenSSH_4.3
[DEBUG] paramiko.transport 1563: Remote version/idstring: SSH-2.0-OpenSSH_4.3
[INFO] paramiko.transport 1563: Connected (version 2.0, client OpenSSH_4.3)
[INFO] paramiko.transport 1563: Connected (version 2.0, client OpenSSH_4.3)
[DEBUG] paramiko.transport 1563: kex algos:[u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'ssh-dss'] client encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] server encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] client mac:[u'hmac-md5', u'hmac-sha1', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] server mac:[u'hmac-md5', u'hmac-sha1', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
[DEBUG] paramiko.transport 1563: kex algos:[u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'ssh-dss'] client encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] server encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] client mac:[u'hmac-md5', u'hmac-sha1', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] server mac:[u'hmac-md5', u'hmac-sha1', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
[DEBUG] paramiko.transport 1563: Kex agreed: diffie-hellman-group1-sha1
[DEBUG] paramiko.transport 1563: Kex agreed: diffie-hellman-group1-sha1
[DEBUG] paramiko.transport 1563: Cipher agreed: aes128-ctr
[DEBUG] paramiko.transport 1563: Cipher agreed: aes128-ctr
[DEBUG] paramiko.transport 1563: MAC agreed: hmac-md5
[DEBUG] paramiko.transport 1563: MAC agreed: hmac-md5
[DEBUG] paramiko.transport 1563: Compression agreed: none
[DEBUG] paramiko.transport 1563: Compression agreed: none
[DEBUG] paramiko.transport 1563: kex engine KexGroup1 specified hash_algo <built-in function openssl_sha1>
[DEBUG] paramiko.transport 1563: kex engine KexGroup1 specified hash_algo <built-in function openssl_sha1>
[DEBUG] paramiko.transport 1563: Switch to new keys ...
[DEBUG] paramiko.transport 1563: Switch to new keys ...
[DEBUG] paramiko.transport 1563: Trying key XXX from /Users/ben/.ssh/id_rsa
[DEBUG] paramiko.transport 1563: Trying key XXX from /Users/ben/.ssh/id_rsa
[DEBUG] paramiko.transport 1563: userauth is OK
[DEBUG] paramiko.transport 1563: userauth is OK
[INFO] paramiko.transport 1563: Authentication (publickey) successful!
[INFO] paramiko.transport 1563: Authentication (publickey) successful!
[DEBUG] pyupdater.uploader 101: Requested uploader: scp
[DEBUG] pyupdater.uploader 101: Requested uploader: scp
[INFO] pyupdater.uploader 118: Plugin: SCP
[INFO] pyupdater.uploader 118: Plugin: SCP
[INFO] pyupdater.uploader 119: Author: Digital Sapphire
[INFO] pyupdater.uploader 119: Author: Digital Sapphire
[DEBUG] paramiko.transport 1095: [chan 0] Max packet in: 32768 bytes
[DEBUG] paramiko.transport 1095: [chan 0] Max packet in: 32768 bytes
[DEBUG] paramiko.transport 1095: [chan 0] Max packet out: 32768 bytes
[DEBUG] paramiko.transport 1095: [chan 0] Max packet out: 32768 bytes
[DEBUG] paramiko.transport 1563: Secsh channel 0 opened.
[DEBUG] paramiko.transport 1563: Secsh channel 0 opened.
[DEBUG] paramiko.transport 1095: [chan 0] Sesch channel 0 request ok
[DEBUG] paramiko.transport 1095: [chan 0] Sesch channel 0 request ok
[DEBUG] paramiko.transport 1095: [chan 0] EOF sent (0)
[DEBUG] paramiko.transport 1095: [chan 0] EOF sent (0)
[DEBUG] pyupdater.uploader 129: versions.gz uploaded successfully
[DEBUG] pyupdater.uploader 129: versions.gz uploaded successfully
[DEBUG] root 369: [Errno 2] No such file or directory: '/Users/ben/Projects/foo/pyu-data/deploy/versions.gz'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/pyupdater/cli/__init__.py", line 367, in upload
    pyu.upload()
  File "/usr/local/lib/python2.7/site-packages/pyupdater/core.py", line 90, in upload
    self.up.upload()
  File "/usr/local/lib/python2.7/site-packages/pyupdater/uploader.py", line 131, in upload
    remove_any(f)
  File "/usr/local/lib/python2.7/site-packages/dsdev_utils/paths.py", line 68, in remove_any
    _remove_any(path)
  File "/usr/local/lib/python2.7/site-packages/dsdev_utils/paths.py", line 65, in _remove_any
    os.remove(path)
OSError: [Errno 2] No such file or directory: '/Users/ben/Projects/foo/pyu-data/deploy/versions.gz'
[DEBUG] root 369: [Errno 2] No such file or directory: '/Users/ben/Projects/foo/pyu-data/deploy/versions.gz'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/pyupdater/cli/__init__.py", line 367, in upload
    pyu.upload()
  File "/usr/local/lib/python2.7/site-packages/pyupdater/core.py", line 90, in upload
    self.up.upload()
  File "/usr/local/lib/python2.7/site-packages/pyupdater/uploader.py", line 131, in upload
    remove_any(f)
  File "/usr/local/lib/python2.7/site-packages/dsdev_utils/paths.py", line 68, in remove_any
    _remove_any(path)
  File "/usr/local/lib/python2.7/site-packages/dsdev_utils/paths.py", line 65, in _remove_any
    os.remove(path)
OSError: [Errno 2] No such file or directory: '/Users/ben/Projects/foo/pyu-data/deploy/versions.gz'
[DEBUG] paramiko.transport 1563: EOF in transport thread
[DEBUG] paramiko.transport 1563: EOF in transport thread

```

--- a/scp_uploader.py
+++ b/scp_uploader.py
@@ -126,7 +126,6 @@ class SCPUploader(BaseUploader):
     def upload_file(self, filename):
         try:
             self.client.put(filename, remote_path=self.remote_dir)
-            os.remove(filename)
             return True
         except Exception as err:
             log.error('Failed to upload file')

